app/background.js

```diff
@@ -3,7 +3,7 @@ const runtime = chrome.runtime;
 const tabs = chrome.tabs;
 const browserAction = chrome.browserAction;
 
-const githubPattern = /^https:\/\/([^\.]+\.)*github\.com/;
+const githubPattern = /^https:\/\/([^\.]+\.)*github*/;
```

Comment:
This regular expression will match `https://abc.githu`, it may not meet your expectation.